newIDE/app/scripts/lib/ExtensionReferenceGenerator.js

```diff
@@ -501,9 +512,9 @@ const generateExtensionReference = extension => {
 
 /**
  * @param {ExtensionReference} extensionReference
- * @param {({extension: gdPlatformExtension, depth: number}) => RawText} generateExtensionHeaderText
- * @param {({extension: gdPlatformExtension}) => RawText} generateExtensionFooterText
- * @returns {Array<RawText>}}}
+ * @param {({extension, depth}) => RawText} generateExtensionHeaderText
+ * @param {({extension}) => RawText} generateExtensionFooterText
```

Comment:
this is the change, it was renaming the variable (rather than typing it)
Plus, it is not aware of gd classes